accounts/service.go

```diff
@@ -41,22 +43,42 @@ func NewService(
 	return &Service{store, km, fc, wp, txs, tes, cfg}
 }
 
-func (s *Service) InitAdminAccount() {
+func (s *Service) InitAdminAccount(ctx context.Context, txService *transactions.Service) error {
 	a, err := s.store.Account(s.cfg.AdminAccountAddress)
 	if err != nil {
 		if !strings.Contains(err.Error(), "record not found") {
-			panic(err)
+			return err
 		}
 		// Admin account not in database
 		a = Account{Address: s.cfg.AdminAccountAddress}
 		err := s.store.InsertAccount(&a)
 		if err != nil {
-			panic(err)
+			return err
 		}
 		AccountAdded.Trigger(AccountAddedPayload{
 			Address: flow.HexToAddress(s.cfg.AdminAccountAddress),
 		})
 	}
+
+	keyCount, err := s.km.InitAdminProposerKeys(ctx)
+	if err != nil {
+		return err
+	}
+
+	if keyCount < s.cfg.AdminProposerKeyCount {
```

Comment:
Create new keys if the current key count is less than the config value

Reply:
Maybe move this logic to a private function:

```go
if keyCount < s.cfg.AdminProposerKeyCount {
  s.addAdminProposalKeys(s.cfg.AdminProposerKeyCount - keyCount)  
}
```

Keeps the code clean and shows the flow of logic

Reply:
Good call, addressed here https://github.com/onflow/wallet-api/pull/13/commits/bb9c04463bda82301e82f56be469352438aa3811